frontend/src/pages/calculatedResults/CalculatedResultsElements.js

```diff
@@ -0,0 +1,80 @@
+import styled from 'styled-components';
+import 'antd/dist/antd.css';
```

Comment:
I guess this one could be imported globally

Reply:
done